score-data-format/src/main/java/io/cloudslang/content/json/services/JsonService.java

```diff
@@ -0,0 +1,154 @@
+package io.cloudslang.content.json.services;
+
+import com.jayway.jsonpath.Configuration;
+import com.jayway.jsonpath.DocumentContext;
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.internal.spi.json.GsonJsonProvider;
+import com.jayway.jsonpath.spi.json.JsonProvider;
+import io.cloudslang.content.json.exceptions.RemoveEmptyElementException;
+import net.minidev.json.JSONArray;
+import net.minidev.json.JSONObject;
+import net.minidev.json.JSONStyle;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Created by Folea Ilie Cristian on 2/3/2016.
+ */
+public class JsonService {
+
+    public String removeEmptyElementsJson(String json) throws RemoveEmptyElementException {
+        String normalizedJson = removeBlanks(json);
+
+        char wrappingQuote = retrieveWrappingQuoteTypeOfJsonMemberNames(normalizedJson);
```

Comment:
maybe remove the removeBlanks() method and just put "json.trim()" as a parameter here